common/inputs.js

```diff
@@ -24,6 +22,22 @@
             }
         }
 
+        /* boolean specific function */
+        var defaultBooleanValues = [true, false];
+
+        // checks for preformat config before returning true/false
+        function formatBoolean(column, value, context) {
+            if (column.display.preformatConfig) {
```

Comment:
I'm going to comment this here, but it's applicable to other places that you've called `formatvalue` or checked `preformatConfig` as well:

- You shouldn't check for `column.display.preformatConfig` and should always call the `formatvalue`. The only thing that we care about is making sure we're showing the same formatted value in the dropdown as we do in record/recordset app.

- The passed `context` is only used for figuring out what the "null" value should be mapped to. I also noticed that it doesn't have to be passed to `formatvalue` function. So I [pushed a change to ermrestjs to use the Reference's context if this parameter is missing](https://github.com/informatics-isi-edu/ermrestjs/commit/8620b51138be64c9e896321f78b17014b6cb1908). And now you can just remove it from all the places that you've called the function. Just `column.formatvalue(value)` should be enough.